packages/suite/src/utils/wallet/coinmarket/coinmarketUtils.ts

```diff
@@ -157,8 +157,19 @@ export const mapTestnetSymbol = (symbol: Network['symbol']) => {
 };
 
 export const getTagAndInfoNote = (quote: { infoNote?: string }) => {
-    if (quote.infoNote?.startsWith('#')) {
-        return { tag: quote.infoNote.substring(1) };
```

Comment:
@MrMebelMan Have you thought about backward compatibility? What if older Suite version receive something like `#badge_text#info_note_text`? Can that happen?

Reply:
@matejkriz I gotta admit that I haven't... Thinking about this now I can't really find a way of handling this... Maybe it would be better to drop this separator-based BS on invity-api side and make the backend return a properly typed object right away, at the same time preserving the original functionality to make it backwards-compatible while avoiding the new format in info notes.

Reply:
Yeah, I don't see any way how to extend this functionality without breaking it for older Suite. I guess the only way is to keep the current api as it is and provide a new one for new Suite versions...